public/index.html

```diff
@@ -44,6 +44,6 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-    <script type="text/javascript" src="https://cdn.fromdoppler.com/doppler-ui-library/build163/js/app.js"></script>
+    <script type="text/javascript" src="https://cdn.fromdoppler.com/doppler-ui-library/build164/js/app.js"></script>
```

Comment:
BTW, probably you want to use build 173 (the last one), and you could test if the fix in the icons that applied @damianmuti is working fine.

Reply:
![image](https://user-images.githubusercontent.com/8861133/55559897-887dc280-56c5-11e9-87fe-3f6c52d6f283.png)

with build 173

Reply:
Mmm, something is wrong, please revert the change. 

@damianmuti could you work on this?

Reply:
Add build 174